triggers/triggers/levels.ts

```diff
@@ -19,9 +20,52 @@ module.exports = <TriggeredCommand> {
             let newLevel = getLevelNumber(level.xp);
             if (newLevel > level.level) {
                 message.reply(`🎉 Congrats! You just leveled up to level ${newLevel}!`);
+                const member = message.member;
+
+                
+                let newRole = member.guild.roles.cache.filter(role =>
+                    role.name.startsWith('Level') &&
+                    getLevelNumberFromRole(role) <= newLevel)
+                    .sort((a:Role, b:Role) => getLevelNumberFromRole(a) - getLevelNumberFromRole(b))
+                    .last();
+                
+                member.roles.cache.forEach(role =>
+                    (newRole.name.startsWith('Level') && role.id != newRole.id) ?
+                        member.roles.remove(role)
+                      : null
+                );
+                
+                member.roles.add(newRole);
```

Comment:
Try pulling this out into `util/levels.ts` and import it from there? It should also run that on every start up to update roles if it fell behind